speech/grpc/transcribe_streaming.py

```diff
@@ -64,7 +64,7 @@ def make_channel(host, port):
     return implementations.secure_channel(host, port, composite_channel)
 
 
-def _audio_data_generator(buff):
+def _audio_data_generator(buff, stoprequest):
```

Comment:
Actually to think of it making both threads stop from the same signal is better.